src/Tizen.Multimedia.Vision/MediaVision/InferenceType.cs

```diff
@@ -30,13 +30,21 @@ public enum InferenceBackendType
         /// <summary>
         /// Tensor Flow Lite backend type
         /// </summary>
-        TFLite
+        TFLite,
+
+        /// <summary>
+        /// ArmNN backend type
+        /// </summary>
+        ArmNN
     }
 
     /// <summary>
     /// Specifies the type of target. It's used for running inference backend.
     /// </summary>
     /// <since_tizen> 6 </since_tizen>
+    /// <remarks>
+    /// Deprecated.(API Level 8) Not used.
```

Comment:
Please use Obsolete attribute instead of <remarks> tag.
```suggestion
    [Obsolete("Please do not use! This will be deprecated in level 8.")]
```

Reply:
Used Obsolete tag.